demisto_sdk/commands/common/tools.py

```diff
@@ -2214,3 +2214,46 @@ def get_pack_dir(path):
         if parts[index] == 'Packs':
             return parts[:index + 2]
     return []
+
```

Comment:
Adding this for future easy code
```suggestion
def listdir_fullpath(dir_name: str) -> List[str]:
    return [os.path.join(dir_name, f) for f in os.listdir(dir_name)]



```